src/components/bcros/Section.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div :data-cy="'section_' + name">
+    <div :data-cy="'header_' + name" class="font-bold text-lg py-2">
```

Comment:
I think it would be better to remove the top padding here and let us apply that where we use this component

Reply:
otherwise we end up with scenarios where we need to apply negative or layered padding on top of this component

Reply:
I will update this before merging the PR